Sources/DeepLearning/Loss.swift

```diff
@@ -27,3 +27,12 @@ public func softmaxCrossEntropy<Scalar: TensorFlowFloatingPoint>(
     logits: Tensor<Scalar>, labels: Tensor<Scalar>) -> Tensor<Scalar> {
     return -(labels * logSoftmax(logits)).mean(alongAxes: 0).sum()
 }
+
```

Comment:
We haven't been doing this well yet, but could you add a Swift documentation comment (`///`-prefixed) describing this function? Maybe something along the lines of:

```
/// Computes the sigmoid cross entropy between logits and labels.
///
/// Logits are one-hot encoded outputs from a neural network. Labels are integer values that correspond to the correct output.
```

or something along those lines?

Reply:
Alright, will do.